pypeit/spectrographs/ldt_deveny.py

```diff
@@ -0,0 +1,399 @@
+            else:
+                decker = "None"
+            return decker
+
+        elif meta_key == 'filter1':
+            """
+            Remove the parenthetical knob position to leave just the filter name
+            """
+            return headarr[0]['FILTREAR'].split()[0].upper()
+
+        else:
+            msgs.error("Not ready for this compound meta for LDT/DeVeny")
+
+    @classmethod
+    def default_pypeit_par(cls):
+        """
+        Return the default parameters to use for this instrument.
+        
+        Returns:
+            :class:`~pypeit.par.pypeitpar.PypeItPar`: Parameters required by
+            all of ``PypeIt`` methods.
+        """
+        par = super().default_pypeit_par()
+
+        # Calibration Parameters
+        # Turn off illumflat -- other defaults OK (as of v1.4.1)
+        set_use = dict(use_illumflat=False)
+        par.reset_all_processimages_par(**set_use)
+
+        # Use median combine (rather than weighted mean) for all cals
+        par['calibrations']['biasframe']['process']['combine'] = 'median'
+        par['calibrations']['arcframe']['process']['combine'] = 'median'
+        par['calibrations']['tiltframe']['process']['combine'] = 'median'
+        par['calibrations']['pixelflatframe']['process']['combine'] = 'median'
+        par['calibrations']['illumflatframe']['process']['combine'] = 'median'
+
+        # For science, standard, and sky frames, combine using weighted mean (Default)
+        #par['calibrations']['standardframe']['process']['combine'] = 'weightmean'
+        #par['calibrations']['skyframe']['process']['combine'] = 'weightmean'
+        #par['scienceframe']['process']['combine'] = 'weightmean'
+
+        # Make a bad pixel mask
+        par['calibrations']['bpm_usebias'] = True
+
+        # Wavelength Calibration Parameters
+        # Include all the lamps available on DeVeny
+        par['calibrations']['wavelengths']['lamps'] = ['NeI', 'ArI', 'CdI', 'HgI']
+        #par['calibrations']['wavelengths']['method'] = 'full_template'
+        # These are changes from defaults from another spectrograph...
+        # TODO: Not sure if we will need to adjust these at some point
+        par['calibrations']['wavelengths']['n_first'] = 3  # Default: 2
+        par['calibrations']['wavelengths']['n_final'] = 5  # Default: 4
+        #par['calibrations']['wavelengths']['nlocal_cc'] = 13  # Default: 11
+        #par['calibrations']['wavelengths']['fwhm']= 3.0  # Default: 4.0
+        par['calibrations']['wavelengths']['fwhm_fromlines'] = True
+        par['calibrations']['wavelengths']['rms_threshold'] = 0.5  # Default: 0.15
+        par['calibrations']['wavelengths']['sigdetect'] = 10.  # Default: 5.0
+        # Needed to address ISSUE #1155 when non-echelle spectrographs use
+        #  the wavelength calibration method `reidentify`.
+        par['calibrations']['wavelengths']['ech_fix_format'] = False
+
+        # Slit-edge settings for long-slit data (slit > 90" long)
```

Comment:
Is this the only slit allowed?  Just checking.

Reply:
The spectrograph has but one slit that nearly completely illuminates the CCD.  There are reflection features in the data at the slit edges (reflective surfaces near the slit jaws) that would frequently cause the edge tracing algorithm to identify narrow slits at the edge of the CCD rather than the actual slit.  The parameters in lines 228 - 230 were those found to allow the edge tracing algorithm to consistently correctly identify the slit.

There are plans to install a decker at some point in the future, and when the opening size(s) are determined, this parameter will be appropriately modified.